Client_Base/src/orsc/PacketHandler.java

```diff
@@ -899,7 +899,7 @@ private void setServerConfiguration() {
 		int sideMenuToggle, inventoryCountToggle, zoomViewToggle;
 		int menuCombatStyleToggle, fightmodeSelectorToggle, experienceCounterToggle;
 		int experienceDropsToggle, itemsOnDeathMenu, showRoofToggle, wantHideIp, wantRemember;
-		int wantGlobalChat, wantSkillMenus, wantQuestMenus, wantQuestStartedIndicator, maxWalkingSpeed;
```

Comment:
Should be able to just copy and paste in the old file, don't see any changes in here that need to be kept

Reply:
Hey Logg, I am working on this one again, and I don't quite understand what you mean by this.

Reply:
The client doesn't seem to actually use showTutorialSkipOption at any time, so it doesn't need to know about it, and the feature should still work. You're handling the disabling of the Tutorial Skip button entirely in packethandler.

Reply:
i.e. you should be able to revert all Client_Base files back to the way they were before & still have identical functionality.

Reply:
> Not sure what changed in `server/inc/sqlite/preservation.db`, but if you're able to, please revert changes on that file.

@Hubcapp Done - I have reverted that file to its state in `develop`.

Reply:
"i.e. you should be able to revert all Client_Base files back to the way they were before & still have identical functionality."
@Hubcapp you're right, I thought those changes to `Client_Base` were needed for the menu to hide the button, but apparently not? Those changes have been reverted.